testdata/scripts/syntax.txt

```diff
@@ -122,6 +124,39 @@ func Test() {
 
 func noop(...interface{}) {}
 
+-- sub/directives.go --
```

Comment:
Was this broken previously?

Reply:
If I understood you correctly, the creation of the file in the directory is broken only if the folder itself already exists. In this case the sub folder does not exist and there is no error.

Reply:
I meant, if this test was broken before this pr (the added functionallity is tested)

Reply:
Yes, I checked. Without this commit, the local name is obfuscated and the export is broken.